validator/client/beacon-api/beacon_api_validator_client.go

```diff
@@ -198,17 +191,24 @@ func (c *beaconApiValidatorClient) WaitForChainStart(ctx context.Context, _ *emp
 	return c.waitForChainStart(ctx)
 }
 
-func (c *beaconApiValidatorClient) StartEventStream(ctx context.Context) error {
-	if c.eventHandler != nil {
-		if err := c.eventHandler.get(ctx, []string{"head"}); err != nil {
-			return errors.Wrapf(err, "could not invoke event handler")
+func (c *beaconApiValidatorClient) StartEventStream(ctx context.Context, topics []string, eventsChannel chan<- *event.Event) {
+	ctx, span := trace.StartSpan(ctx, "validator.beaconApiClient.StartEventStream")
+	defer span.End()
```

Comment:
If this method is meant to start a stream and block until the stream is done (forever basically) then this isn't a meaningful thing to measure

```suggestion
```